kotlin/mobilesdkrs/build.gradle

```diff
@@ -46,8 +46,10 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
 
-    implementation "net.java.dev.jna:jna:5.13.0@aar"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4"
+    implementation 'net.java.dev.jna:jna:5.13.0@aar'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.9.0-RC.2'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.9.0-RC.2'
```

Comment:
TODO: Revert changes to `1.6.4`. See previous comments for reasoning. (i.e. support more phones/users)